src/infra/dataverse/sparql/sparqlGateway.ts

```diff
@@ -58,16 +58,23 @@ export const sparqlGateway: DataversePort = {
         ?id rdf:type ?type .
         ?type rdf:type owl:Class .
         ?metadata core:describes ?id .
-        ?metadata core:hasTitle ?title .
-        FILTER langMatches(lang(?title),'${language}')
+        OPTIONAL { 
+          ?metadata core:hasTitle ?title .
+          FILTER langMatches(lang(?title),'${language}')
+        }
+        OPTIONAL { 
+          ?metadata core:hasTitle ?title .
+        }
         ?metadata core:hasPublisher ?publisher .
         OPTIONAL { 
-          ?topic skos:prefLabel ?filteredPrefLabel .
-          FILTER langMatches(lang(?filteredPrefLabel),'${language}')
+          ?topic skos:prefLabel ?prefLabel .
+          FILTER langMatches(lang(?prefLabel),'${language}')
+        }
+        OPTIONAL { 
+          ?topic skos:prefLabel ?prefLabel .
         }
-        ?topic skos:prefLabel ?fallbackPrefLabel .
       }
-      ORDER BY ?title
+      ORDER BY asc(str(?title))
```

Comment:
why is it required?

Reply:
Otherwise, the query will first group by the language desired by the user, then sort. 
As a practical consequence, the results concerning the fallback language come afterwards and are not sorted.
This is the only way to have the sorting performed on the title value, whatever the language.